templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>It may be worth mentioning that the multipliers are on the current_channel_name and not on the entire matrix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>